toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -101,7 +101,11 @@ done
 [[ -z "${repo_path}" ]] && repo_path=${script_dir} && repo_path=${repo_path%'/toolkit'*}
 [[ ! -d "${repo_path}" ]] && { print_error " Directory ${repo_path} does not exist"; exit 1; }
 [[ -z "${mode}" ]] && mode="build"
-[[ -z "${version}" ]] && version="2.0"
+[[ -z "${version}" ]] && version="3.0"
+# TODO: Remove when PMC is available for 3.0
+if [[ "${version}" == "3.0" ]]; then
+    [[ -z "${DAILY_BUILD_ID}" ]] && DAILY_BUILD_ID="3-0-$(date -d "2 day ago" '+%Y%m%d')" #latest available
```

Comment:
Does this just always take the build from 2 days ago? We have an LKG.json file that can be used